internal/apilogic/handle_request.go

```diff
@@ -23,6 +24,11 @@ func BuildHandleRequest(req *http.Request) (*handle.Request, error) {
 		handleRequest.Mode = mode
 	}
 
+	// exposeAllKeys default true for flags route
+	if strings.Contains(req.URL.String(), "/flags") {
```

Comment:
Hum, so if the url is `/flags?exposeAllKeys=toto`, I feel like the handleRequest.ExposeAllKeys flag will be false. Is that the expected behavior ?

Reply:
Also, I guess you thought about it, but there was no way to do it without hardcoding this "if", right ?

Reply:
It is possible I I refactor the BuildHandleRequest method, car for the moment il takes only the Request as an argument and golang does not allow to update the request from the controller :/

Reply:
If `/flags?exposeAllKeys=toto`, yes it will be false but it should not be implemented also ?